LoginForm/Model1.Context.tt

```diff
@@ -2,7 +2,7 @@
 <#@ include file="EF6.Utility.CS.ttinclude"#><#@
  output extension=".cs"#><#
 
-const string inputFile = @"LoginModel.edmx";
+const string inputFile = @"Model1.edmx";
```

Comment:
Model1 is a bad name. it should be ToDoAppSystemModel
Also the Model should be in the folder Model.